test/integration/verify/controls/google_storage_object_acl.rb

```diff
@@ -0,0 +1,19 @@
+title 'Test single GCP storage object ACL'
+
+gcp_project_id = attribute(:gcp_project_id, default: '', description: 'The GCP project identifier.')
+gcp_storage_bucket_object = attribute(:gcp_storage_bucket_object, default: '', description: 'The GCP bucket with objects.')
+gcp_storage_bucket_object_name = attribute(:gcp_storage_bucket_object_name, default: '', description: 'The GCP bucket object name.')
+gcp_enable_privileged_resources = attribute(:gcp_enable_privileged_resources,default:0,description:'Flag to enable privileged resources requiring elevated privileges in GCP.')
+
+control 'gcp-storage-object-acl-1.0' do
+
+  only_if { gcp_enable_privileged_resources.to_i == 1 }
```

Comment:
what's the benefit of `to_i`?

Reply:
we're comparing integers - the config comes through as a string

Reply:
Ah - at a glance looked like boolean to integer

Reply:
it's funny as I'd have preferred to use booleans but tf converts true/false values to "0" and "1" -  https://www.terraform.io/docs/configuration/variables.html